static/src/javascripts/projects/common/modules/atoms/youtube.js

```diff
@@ -373,7 +365,7 @@ const onPlayerReady = (
     const iFrameBehaviourConfig = getIFrameBehaviourConfig(iframe);
     const iFrameBehaviour = getIFrameBehaviour(iFrameBehaviourConfig);
     if (iFrameBehaviour.mutedOnStart) {
-        muteIFrame(iframe);
+        event.target.mute();
```

Comment:
I think `event.target` gets assigned to `youtubePlayer` [a bit further up](https://github.com/guardian/frontend/pull/25199/files#diff-ef828fcf95131da1b9fcb893ba39b1e82b00da5dd2deecda295303148ff4cc95R350)
```suggestion
        youtubePlayer.mute();
```

Reply:
Nice spot, have updated the PR with this.